src/helpers/issue.ts

```diff
@@ -3,6 +3,37 @@ import { getBotConfig, getBotContext, getLogger } from "../bindings";
 import { AssignEvent, Comment, IssueType, Payload } from "../types";
 import { checkRateLimitGit } from "../utils";
 
+export const getAllIssueEvents = async () => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  if (!payload.issue) return;
+  try {
+    // Fetch issue events
+    const { data: events } = await context.octokit.issues.listEvents({
```

Comment:
you need to use pagination to retrieve all events, you can look at other functions as an example

Reply:
whats pagination

Reply:
what shoul be page value

Reply:
what do you mean?

Reply:
There was a per-page parameter and a page parameter, I just set the page parameter to one because idk

Reply:
just make pagination like other functions

Reply:
example: [getAllIssueAssignEvents](https://github.com/ubiquity/ubiquibot/blob/5d10bda43e832717d7daba5949543e68f3dffbb8/src/helpers/issue.ts#L257)

Reply:
Sorry about it I was busy with school